modules/settings/Settings.js

```diff
@@ -118,7 +118,6 @@ export default {
     },
     setLanguage: function (lang) {
         language = lang;
-        jitsiLocalStorage.setItem("language", lang);
```

Comment:
Why is this removed? I guess because the local storage backend takes care, but then why do we load it from there on line 27?